BugzillaQuery.class.php

```diff
@@ -222,28 +199,36 @@ public function _fetch_by_options() {
         }
 
         if(!empty($this->options['include_fields'])) {
-            $this->options['include_fields'] = implode(",", $this->options['include_fields']);
+            $this->options['include_fields']
+                = implode(",", $this->options['include_fields']);
         }
 
         // Add the requested query options to the request
-        $url = $request->getUrl();
-        $url->setQueryVariables($this->options);
+        $ua = MWHttpRequest::factory( $this->url . '?'
+                                           . http_build_query( $this->options ),
```

Comment:
You may remove the block before from line 186 working with `$this->options` and use `$this->rebased_options()` here (which should take care of that).